pages/language/guides/bounced.mdx

```diff
@@ -2,11 +2,11 @@ import { Callout } from 'nextra-theme-docs'
 
 # Bounced messages
 
-When contract sends a message with a flag bounce set to true, than if the message wasn't processed properly, it would bounce back to the sender. This is useful when you want to make sure that the message was processed properly and if not - revert the changes.
+When a contract sends a message with a flag bounce set to true, then if the message wasn't processed properly, it would bounce back to the sender. This is useful when you want to make sure that the message was processed properly and if not - revert the changes.
 
 ## Caveats
 
-Currently in TON bounced messages have only 224 usable data bits in the message and no references. This means that you can't recover much of a data from the bounced message. This is a limitation of the TON blockchain and will be fixed in the future. Tact helps you to check if your message fits the limit and if not - it will suggest to use a special type modifier `bounced<T>` for receiver that would construct a partial representation that fits into required limits.
+Currently in TON bounced messages have only 224 usable data bits in the message and no references. This means that you can't recover much of the data from the bounced message. This is a limitation of the TON blockchain and will be fixed in the future. Tact helps you to check if your message fits the limit and if not - it will suggest to use of a special type modifier `bounced<T>` for the receiver that would construct a partial representation that fits into the required limits.
```

Comment:
```suggestion
Currently in TON bounced messages have only 224 usable data bits in the message and no references. This means that you can't recover much of the data from the bounced message. This is a limitation of the TON blockchain and will be fixed in the future. Tact helps you to check if your message fits the limit and if not - it will suggest to use a special type modifier `bounced<T>` for the receiver that would construct a partial representation that fits into the required limits.
```